File: C Programs/Problems on String/Accept string & check whether it contains vowels in it or not.c
/* Write a program which accept string from user and check whether it contains vowels in it or not.

Input : “marvellous”
Output : TRUE

Input : “Demo”
Output : TRUE

Input : “xyz”
Output : FALSE

*/

#include<stdio.h>

typedef int BOOL;
#define TRUE 1
#define FALSE 0

BOOL ChkVowel(char str[])
{

    int i=0;
    while(str[i]!='\0')
     {
        if((str[i]=='A') || (str[i]=='a') || (str[i]=='E') || (str[i]=='e') || (str[i]=='I') || (str[i]=='i') || (str[i]=='O') || (str[i]=='o') || (str[i]=='U') || (str[i]=='u'))
          {
             return TRUE;
          }
        i++;
    }
}

int main()
{
    char arr[30];
    BOOL bRet = 0;

    printf("Enter string: \n");
    scanf("%[^'\n']s",arr);

    bRet = ChkVowel(arr);

    if(bRet == TRUE)
      {
         printf("Contains Vowel");
      }
    else
      {
         printf("There is no Vowel");
      }

    return 0;
}

File: Java Programs/Problems on String/Accept string & check whether it contains vowels in it or not.java
/* Write a java program which accept string from user and check whether it contains vowels in it or not.

Input : marvellous
Output : TRUE

Input : Demo
Output : TRUE

Input : xyz
Output : FALSE 

*/

import java.lang.*;
import java.util.*;

class DemoString
{
    boolean CntVowels(String str)
    {
        int i=0;
        boolean bAns=false;
        char arr[]=str.toCharArray();
        int size=arr.length;
        
        while(i<size)
       
       {
           if((arr[i]=='A')||(arr[i]=='a')||(arr[i]=='E')||(arr[i]=='e')||(arr[i]=='I')||(arr[i]=='i')||(arr[i]=='O')||(arr[i]=='o')||(arr[i]=='U')||(arr[i]=='u'))
           {
               bAns=true;
               break;
           }
		          i++;
       }
           
          return bAns;
      }
}

class Demo
        {
             public static void main(String arg[])
             {
                 Scanner sobj=new Scanner(System.in);
                 
                 DemoString dobj=new DemoString();
                 
                 System.out.println("Enter String: ");
                 String str=sobj.nextLine();
                 
                 boolean bRet=dobj.CntVowels(str);

                 if(bRet==true)
                 {
                    System.out.println("Vowels are present");
                 }
               else
               {
                  System.out.println("Vowels are not present");
               }
            }
        }
